src/customers/service/customers.service.ts

```diff
@@ -14,7 +19,21 @@ export class CustomersService {
     return customer;
   }
 
-  createCustomer(customer: Customer): Promise<Customer> {
-    return this.customersRepository.create(customer);
+  async getCustomersByUsername(username: string) {
+    return await this.customersRepository.findOneByUsername(username);
+  }
+
+  // async createCustomer(customer: Customer): Promise<Customer> {
+  //   return await this.customersRepository.create(customer);
+  // }
```

Comment:
Do not forget to remove the commented code, unused imports, maybe some `console.log`s used for testing, etc.

Reply:
done